manim/mobject/geometry.py

```diff
@@ -710,6 +710,20 @@ def set_opacity(self, opacity, family=True):
 
 
 class DashedLine(Line):
+    """A dashed Line.
+
+    Examples
+    --------
+    .. manim:: DashedLineExample
+        :save_last_frame:
+
+        class DashedLineExample(Scene):
+            def construct(self):
+                l = DashedLine(config.frame_width/2*LEFT, 4*RIGHT)
+                self.add(l)
```

Comment:
https://www.python.org/dev/peps/pep-0008/#names-to-avoid -- `l` should be avoided as a variable name.

```suggestion
                dashed_line = DashedLine(config.frame_width/2*LEFT, 4*RIGHT)
                self.add(dashed_line)
```